src/test/kotlin/com/example/realworldkotlinspringbootjdbc/presentation/FavoriteControllerTest.kt

```diff
@@ -125,4 +127,108 @@ class FavoriteControllerTest {
             }
         }
     }
+
+    class Unfavorite {
+        private val requestHeader = "dummy-authorize"
+        private val pathParamSlug = "dummy-slug"
+        val dummyRegisteredUser = RegisteredUser.newWithoutValidation(
+            UserId(1),
+            Email.newWithoutValidation("dummy@example.com"),
+            Username.newWithoutValidation("dummy-name"),
+            Bio.newWithoutValidation("dummy-bio"),
+            Image.newWithoutValidation("dummy-image"),
+        )
+
+        data class TestCase(
+            val title: String,
+            val useCaseExecuteResult: Either<UnfavoriteUseCase.Error, CreatedArticle>,
+            val expected: ResponseEntity<String>,
+        )
+
+        @TestFactory
+        fun test(): Stream<DynamicNode> {
+            return Stream.of(
+                TestCase(
+                    title = "成功: UnfavoriteUseCase が Unit を返す場合、200 レスポンスを返す",
+                    useCaseExecuteResult = CreatedArticle.newWithoutValidation(
+                        id = ArticleId(1),
+                        title = Title.newWithoutValidation("dummy-title"),
+                        slug = Slug.newWithoutValidation("dummy-slug"),
+                        body = ArticleBody.newWithoutValidation("dummy-body"),
+                        createdAt = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX").parse("2022-01-01T00:00:00+09:00"),
+                        updatedAt = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssXXX").parse("2022-01-01T00:00:00+09:00"),
+                        description = Description.newWithoutValidation("dummy-description"),
+                        tagList = listOf(
+                            Tag.newWithoutValidation("dummy-tag1"),
+                            Tag.newWithoutValidation("dummy-tag2")
+                        ),
+                        authorId = UserId(1),
+                        favorited = false,
+                        favoritesCount = 0
+                    ).right(),
+                    expected = ResponseEntity(
+                        """{"article":{"title":"dummy-title","slug":"dummy-slug","body":"dummy-body","createdAt":"2021-12-31T15:00:00.000Z","updatedAt":"2021-12-31T15:00:00.000Z","description":"dummy-description","tagList":["dummy-tag1","dummy-tag2"],"author":"1","favorited":false,"favoritesCount":0}}""",
+                        HttpStatus.valueOf(200)
+                    )
+                ),
+                TestCase(
+                    title = "失敗: UnfavoriteUseCase が「不正なSlug（InvalidSlug）」エラーを返す場合、404 レスポンスを返す",
```

Comment:
[nits]

失敗 →準正常系

でお願いいたします 🙇 

```suggestion
                    title = "準正常系: UnfavoriteUseCase が「不正なSlug（InvalidSlug）」エラーを返す場合、404 レスポンスを返す",
```

Reply:
こちら修正しました。

https://github.com/sunakan/realworld-kotlin-springboot-jdbc/pull/111/commits/01c4f67403aa4eaa972fc5584a109ba7997c4dba